src/engine/ruleset/decoders/f5/f5-apm-own.yml

```diff
@@ -0,0 +1,451 @@
+  - check: event.provider==01260009
+    logpar:
+      - ~content: <~/ignore/ >:<~>:<~/ignore/:> Connection error<~/ignore/:><~tmp.rsa_event_desc>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - rsa.misc.log_session_id : $~tmp.log_session_id
+      - rsa.internal.event_desc : $~tmp.rsa_event_desc
+
+
+  # 01490166 || 01490167 || 0149016a
+  # nisiut: 01490166: :rumwri: Current snapshot ID: velill retrieved from session db for access profile: ore
+  # isqu: 01490167: :uis: Current snapshot ID: idolore updated inside session db for access profile: onse
+  # February 2019/02/17 03:30:32 etconse medium uinesci: 0149016a: :otamr: Initiating snapshot creation: tsed for access profile: rExc
+  - check: event.provider==01490166 OR event.provider==01490167 OR event.provider==0149016a OR event.provider==0149016b
+    logpar:
+      - ~content: <~/ignore/ >:<~>:<~/ignore/:> Current snapshot ID:<~>
+      - ~content: <~/ignore/ >:<~>:<~/ignore/:> <~/literal/Initiating>?<~/literal/Completed> snapshot<~>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+
+  # 01420002
+  # February 2018/02/24 19:26:15 oeiusmo very-high cusanti[5019]: 01420002: : AUDIT - pid=4996 user=rem folder=tseddoei module=teursint status=success cmd_data=remagnaa
+  - check: event.provider==01420002
+    logpar:
+      - ~content: :<~/ignore/ >AUDIT - pid=<~tmp.process.parent.id> user=<user.name> folder=<~tmp.file.directory> module=<~> status=<~tmp.rsa.misc.result> cmd_data=<~tmp.rsa.db.index>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - process.parent.id : $~tmp.process.parent.id
+      - file.directory : $~tmp.file.directory
+      - rsa.misc.result : $~tmp.rsa.misc.result
+      - rsa.db.index : $~tmp.rsa.db.index
+
+  # 01490000
+  # July 2019/07/10 01:56:14 tatemse low vitae[72]: 01490000: :samvolu: dip
+  - check: event.provider==01490000
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~tmp.rsa.internal.event_desc>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - rsa.internal.event_desc : $~tmp.rsa.internal.event_desc
+
+  # 01490004
+  # November 2017/11/02 11:05:41 iquip very-high sedquian[4212]: 01490004: :etdolore: magnaa: Executed agent 'sumquiad', return value iusmodt
+  - check: event.provider==01490004
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~>'<~tmp.network.application>', return value <~tmp.rsa.misc.result_code>
+    map:
+      - rsa.internal.messageid : $event.provider
+      - event.code : $event.provider
+      - rsa.misc.result_code : $~tmp.rsa.misc.result_code
+
+  # 01490005
+  # February 2017/02/03 21:16:50 sBono high equ[4808]: 01490005: :amvo: siuta: Following rule urmagn from item dquia to ending temporin
+  - check: event.provider==01490005
+    logpar:
+      - ~content: :<~>:<~/ignore/ ><~tmp.rsa.misc.log_session_id>:<~>
```

Comment:
there are sine patterns repeated on several cases like 
`:<~>:<~/ignore/ ><~tmp.rsa.misc.log_session_id>`
moving it to a definition could be a good way of improving readability.